### PR TITLE
add type hint for `minio/credentials/credentials.py`

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

File: minio/credentials/credentials.py [hunk 1]
```diff
@@ -43,23 +53,24 @@ def __init__(
...
         """Get session token."""
         return self._session_token
 
-    def is_expired(self):
+    def is_expired(self) -> bool:
         """Check whether this credentials expired or not."""
         return (
             self._expiration < (datetime.utcnow() + timedelta(seconds=10))
-            if self._expiration else False
+            if self._expiration
```
Comment (balamurugana) [inline minio/credentials/credentials.py:74]: Avoid unnecessary diffs

**Review by balamurugana — CHANGES_REQUESTED** [review]

File: minio/credentials/credentials.py (hunk 1 above)
Comment (trim21) [reply minio/credentials/credentials.py]: done.

have you consider add [black](https://github.com/psf/black) as formatter？

**Review by balamurugana — APPROVED** [review]

File: minio/credentials/credentials.py (hunk 1 above)
Comment (balamurugana) [reply minio/credentials/credentials.py]: We can unless it fixes real issues.

Comment (trim21) [reply minio/credentials/credentials.py]: it help avoid unnecessary diffs like this...

Comment (balamurugana) [reply minio/credentials/credentials.py]: You could disable `black` for `minio-py`

Comment (trim21) [reply minio/credentials/credentials.py]: I'm using formatter of pycharm, not black here...

**Comment by trim21** [comment]
@balamurugana does this PR still need more reviewers?

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**